scripts/provisioning/s3_setup

```diff
@@ -324,7 +355,7 @@ then
 
     cmd_out=$(ldapsearch -b "o=s3-background-delete-svc,ou=accounts,dc=s3,dc=seagate,dc=com" -x -w $ldappasswd -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldap://) || echo ""
     if [[ $cmd_out == *"No such object"* ]];then
-        echo "ERROR: failed to find s3background delete account, exit."
+        echo "ERROR: failed to find s3background delete account, sanity test failed, exiting."
```

Comment:
is it sanity test ?

Reply:
For time being we have kept above "ldapsearch" as sanity test, as our santiy test script: "s3-santiy-test.sh" needs to be fixed.